core/resources/shaders/polyline.vs

```diff
@@ -69,7 +70,7 @@ void main() {
         vec4 extrude = UNPACK_EXTRUSION(a_extrude);
         float width = extrude.z;
         float dwdz = extrude.w;
-        float dz = u_map_position.z - abs(u_tile_origin.z);
+        float dz = u_map_position.z - u_tile_origin.w;
```

Comment:
It is currently correct to use the data zoom here because width is evaluated based on the data zoom. See the comments here: https://github.com/tangrams/tangram/pull/266